compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cpp

```diff
@@ -24,7 +24,7 @@
 // Must go after undef _FILE_OFFSET_BITS.
 #include "sanitizer_platform.h"
 
-#if SANITIZER_LINUX || SANITIZER_APPLE || SANITIZER_HAIKU
```

Comment:
Maybe sanitizer_platform_limits_aix.cpp ?

Reply:
Why though? AIX is a POSIX platform and the logic (and structure) here seems to be common except for the usual type/structure size/layout differences.